table/internal_iterator.h

```diff
@@ -130,6 +132,7 @@ class InternalIteratorBase : public Cleanable {
       Prev();
     }
   }
+  bool is_mutable_ = true;
```

Comment:
Nit: Can this be default initialized in the constructor for sake of consistency?